package.json

```diff
@@ -21,8 +21,8 @@
     "docs": "esdoc",
     "test": "yarn test:unit",
```

Comment:
I kinda think test should just be everything. We've got a bit of a weird split between unit/integration right now and especially if the default is mocked, I feel like the test options should be "run all the tests" and "run all the tests unmocked"

Reply:
yeah, good call 👍